fcli-ssc/src/main/java/com/fortify/cli/ssc/role/cli/cmd/SSCRoleGetCommand.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * (c) Copyright 2021 Micro Focus or one of its affiliates
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a 
+ * copy of this software and associated documentation files (the 
+ * "Software"), to deal in the Software without restriction, including without 
+ * limitation the rights to use, copy, modify, merge, publish, distribute, 
+ * sublicense, and/or sell copies of the Software, and to permit persons to 
+ * whom the Software is furnished to do so, subject to the following 
+ * conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY 
+ * KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE 
+ * WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR 
+ * PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE 
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, 
+ * DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF 
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN 
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS 
+ * IN THE SOFTWARE.
+ ******************************************************************************/
+package com.fortify.cli.ssc.role.cli.cmd;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fortify.cli.ssc.rest.SSCUrls;
+import com.fortify.cli.ssc.rest.cli.cmd.AbstractSSCGetCommand;
+import com.fortify.cli.ssc.role.cli.mixin.SSCRoleResolverMixin;
+import io.micronaut.core.annotation.ReflectiveAccess;
+import kong.unirest.GetRequest;
+import kong.unirest.UnirestInstance;
+import picocli.CommandLine.Mixin;
+import picocli.CommandLine.Command;
+
+@ReflectiveAccess
+@Command(name = "get")
+public class SSCRoleGetCommand extends AbstractSSCGetCommand {
+    @Mixin
+    private SSCRoleResolverMixin.PositionalParameter role;
+
+    protected GetRequest generateRequest(UnirestInstance unirest) {
+        return unirest.get(SSCUrls.ROLE(role.getRoleId(unirest)));
+    }
+
+    @Override
+    protected JsonNode generateOutput(UnirestInstance unirest) {
+        return unirest.get(SSCUrls.ROLE(role.getRoleId(unirest)))
+                .asObject(ObjectNode.class).getBody();
+    }
+
+    @Override
+    protected boolean isOutputWrappedInDataObject() { return true; }
```

Comment:
This is the default, so no need to override

Reply:
Done.